### DISCUSSION TITLE
position: sticky header on web like Twitter?

### DISCUSSION THREAD

**Discussion by ghost** [react-navigation/react-navigation#9532] [Q&A]
Is it possible to configure stack navigator headers to be positioned with sticky on web?

**Comment by ghost** [comment]
It looks like adding `position: sticky` to `headerStyle` doesn't work: it looks like it needs to be applied to the root `Animated.View` component, or otherwise accounted for. Has anyone else tried doing this?

**Reply by raajnadar** [reply]
Keep the stack header outside scrollview

**Reply by ghost** [reply]
The behavior isn't quite the same: wouldn't that create a scrollable area that wouldn't contain the header, rather than being able to use the scroll of the parent (e.g. the body)? I'd like to use the body scroller and just have the header sticky above it, if possible.
